pkgs/by-name/ze/zerotier-systemd-manager/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "zerotier-systemd-manager";
+  version = "0.4.0"; # Adjust accordingly
+
+  src = fetchFromGitHub {
+    owner = "zerotier";
+    repo = "zerotier-systemd-manager";
+    rev = "v${version}";
+    sha256 = "sha256-vq6AqrA9ryzLcLEsPD2KbBZhF5YjF48ErIWb8e3b9JI=";
+  };
+
+  vendorHash = "sha256-40e/FFzHbWo0+bZoHQWzM7D60VUEr+ipxc5Tl0X9E2A=";
+
+  meta = with lib; {
```

```suggestion
  meta = {
```

https://github.com/NixOS/nixpkgs/issues/371862

---

Is required for `lib.maintainers` and `lib.licenses`